functions/exercises.py

```diff
@@ -171,10 +179,14 @@ def check_prime(number):
     :param number:
     :return:
     """
-    # INSERT YOUR CODE HERE
-    #
-    # Run this command to test your implementation:
-    #
+    if number <= 1:
+        return False
+
+    for n in range(2,int(math.ceil(number**0.5))):

```

Comment:
```suggestion
    for n in range(2,math.ceil(number**0.5)):
```
`math.ceil` is supposed to give you an integer already right?

Can you explain a bit your line of thought?
Why do we need `number**0.5`? 
and why don't you use `math.sqrt` since you have already imported that module?

Reply:
Yes you are right, I remember I had some kind of error using that because of the integer type but maybe was my logic that was flawed and I ended up leaving it like that.

Also we just need to check if its divisible until the square root of its number to see if the number is prime, the reason is as follows:

If a number n is not a prime, it can be factored into two factors a and b:

n = a * b

Now a and b can't be both greater than the square root of n, since then the product a * b would be greater than sqrt(n) * sqrt(n) = n. Therefore if there isnt any factor until the square root of n , the number is prime